src/Prelude.agda

```diff
@@ -76,3 +78,11 @@ syntax ∃₂-syntax (λ x y → C) = ∃₂[ x , y ] C
 import Data.Nat.Literals as ℕ
 
 instance Number-ℕ = ℕ.number
+
+-- (pseudo)equality (for Maybe)
+_≡?_ : {A : Type} → Maybe A → Maybe A → Type
```

Comment:
I find this symbol a bit confusing since `?` is everywhere else? used for decidability purposes.

I'd suggest to rename it to one of: ≊ ≋ ≅

Reply:
I like the first two equality symbols you suggest, but the last is overused.

Reply:
I've seen the symbol ≃ used for Kleene equality (https://en.wikipedia.org/wiki/Kleene_equality).

Since the notion of equality  we use is even weaker, I'm leaning towards ≊. We can think of every ~ as weakening the notion of equality.

Reply:
I've seen ≛ and ≜ being used to denote 'equal by definition'.

Reply:
I've see `≜` and `:=` (and sometimes an equal with a little `def` on top of it) used for "equal by definition," but I've never see `≛` used for that purpose.  I think we should avoid using `≜` for anything other than definitional equality... and even for that purpose I don't like it.  I agree with Andre that there so are many equality symbols to choose from, so we can probably find one that avoids using tildes.  Having said that, I have no problem with using tildes for this.

Tildes are used all over the place and for so many different notions of equality that one can never assume one knows in advance what it means.  I think that's actually what we want here---the reader needs to look at the definition to know what we mean by whatever symbol we choose for this "equality."

Reply:
> I've seen ≛ 

The symbol displays differently in a code block:
≛ vs `≛`

I definitely have not seen ≛ used for eq. by def.

Reply:
I agree that a simple `~` will not lead to possible confusions with equality and will force people to look it up.